doc/decisions/DR016-Related-entities-result-paging.md

```diff
@@ -0,0 +1,352 @@
+# DR016 Related entities result paging
```

Comment:
[fix] It would be good to call out that this is a more general mechanism for any one-to-many type methods, eg we would also want to apply it to `entityVersions`.